test/cli/autogen-subclasses/autogen-subclasses.out

```diff
@@ -1,7 +1,8 @@
 --- autogen-subclasses ---
 module Opus::Mixin
- Opus::Mixed
```

Comment:
Is this a backwards compatible change?
If not, please do a careful rollout and try to minimize the window of non-bumpable sorbet(in particular because we're currently hunting an infra error that has low rate of failures to propagate sorbet cache)